spec/models/inventory_spec.rb

```diff
@@ -52,11 +52,13 @@
   end
 
   describe '#save' do
-    subject {
-      create(:inventory)
-    }
+    let(:owner) { FactoryGirl.create(:user) }
+    subject { FactoryGirl.create(:inventory, owner: owner) }
```

Comment:
If the factory fails to save the entity, then everywhere that we make use of this factory will also fail for the same reason.  It's more for us to be sure that we isolate failures, but for now it will suffice.